readme.md

```diff
@@ -13,31 +13,70 @@ npm install @cycle/react-native
 
 ```js
 import xs from 'xstream';
-import {run} from '@cycle/run';
-import {makeReactNativeDriver, TouchableOpacity, View, Text} from '@cycle/react-native';
+import {setup} from '@cycle/run';
+import {
+  makeReactNativeDriver,
+  View,
+  TouchableHighlight,
+  Text,
+} from '@cycle/react-native';
+import {name as appName} from './app.json';
+
+const styles = {
+  view: {
+    justifyContent: 'center',
+    alignItems: 'center',
+    height: '100%',
+  },
+  button: {
+    backgroundColor: '#191815',
+    borderWidth: 1,
+    justifyContent: 'center',
+    alignItems: 'center',
+    padding: 10,
+    width: 150,
+    borderRadius: 10,
+  },
+  buttonText: {
+    color: 'white',
+  },
+};
```

Comment:
For the purposes of a readme example, we don't need styles.

Reply:
I’d recommend keeping the `view` container style , since on modern devices the counter touchable goes behind the statusbar / notch / etc. 

i’ll remove the other styles